src/ethereum/curve/contracts.ts

```diff
@@ -5,6 +5,9 @@ import poolTokenV2Abi from "./abi/CurvePoolTokenV2.json";
 import poolTokenV3Abi from "./abi/CurvePoolTokenV3.json";
 import zapAbi from "./abi/CurveZap.json";
 import metaZapAbi from "./abi/CurveMetaZap.json";
+import metapoolFactory from "./abi/CurveMetapoolFactory.json";
```

Comment:
All of the imports should be `CamelCased` and matching their JSON filenames:

`import CurvePoolTokenV3Abi from "./abi/CurvePoolTokenV3.json;`